packages/react-native-worklets/package.json

```
@@ -66,6 +66,8 @@
     "android",
     "apple",
     "Common",
+    "scripts",
```

Let's not potentially add dev-only scripts.

```suggestion
    "scripts/worklets_utils.rb",
```